plugins/arcgis-sde-data-common/src/index.ts

```diff
@@ -50,7 +50,7 @@ export interface DomainValue {
 
 export interface ArcGISSDEDomainEntity extends Entity {
   apiVersion: 'geoportia.se/v1alpha1';
-  kind: 'Table';
+  kind: 'ArcGISDomain';
```

Comment:
As with tables, domains (and domain values) should be more generic, as they also exist in other databases. The `dialect`-pattern can be reused here.